src/cmd/health.js

```diff
@@ -108,14 +130,81 @@ async function get_service_memory_usage() {
 }
 
 
+function get_root_fs_context(config_root_backend) {
+  return {
+      uid: process.getuid(),
+      gid: process.getgid(),
+      warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
+      backend: config_root_backend
+  };
+}
+
+async function get_bucket_storage_status(config_root_backend) {
+  return await get_storage_status(config_root_backend, 'bucket');
+}
+
+async function get_account_storage_status(config_root_backend) {
+  return await get_storage_status(config_root_backend, 'account');
+}
+
+async function get_storage_status(config_root_backend, type) {
+  const fs_context = get_root_fs_context(config_root_backend);
+  const entries = await nb_native().fs.readdir(fs_context, get_config_path(config_root_backend, type));
+  const config_files = entries.filter(entree => !isDirectory(entree) && entree.name.endsWith('.json'));
+  const missing_storages = [];
+  try {
+    for (const config_file of config_files) {
+      const config_file_path = path.join(get_config_path(config_root_backend, type), config_file.name);
+      const raw_data = await fs.promises.readFile(config_file_path);
+      const config_data = JSON.parse(raw_data.toString());
+      if (config_data) {
+        const storage_path = type === 'bucket' ? config_data.path : config_data.nsfs_account_config.new_buckets_path;
+        const dir_stat = await fs_utils.file_exists(storage_path);
+        if (!dir_stat) {
+            console.error(`Error: Storage path should be a valid dir path`, storage_path);
+            const missing_storage = {
+              name: config_data.name,
+              storage_path: config_data.path ? config_data.path : config_data.nsfs_account_config.new_buckets_path,
+            };
+            missing_storages.push(missing_storage);
+        }
+      } else {
+          console.error('Could not parse config file: ', config_file);
+      }
+    }
+  } catch (err) {
+      console.error('Error while accessing the storage: ', err);
+  }
+  return missing_storages;
+}
+
+function get_config_path(config_root_backend, type) {
+  return path.join(config_root_backend, type === 'bucket' ? '/buckets' : '/accounts');
+}
+
+function isDirectory(ent) {
```

Comment:
this function already exists in namespace_fs, can you move it to native_fs_utils.js and re-use it?